lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -452,7 +452,8 @@ def send_bulk(data)
       response = client.bulk body: data
       if response['errors']
         @error.handle_error(response)
-        log.error "Could not push log to Elasticsearch: #{response}"
+        log.error "Could not push log to Elasticsearch with Bulk Request.  Enable debug to see full response."
```

Comment:
Can we move these messages into the error handler?

Reply:
`Enable debug to see full response.` is not good for users.
Can we use `Use @log_level debug to see full response.` instead?

Reply:
@cosmo0920 I don't understand what you are suggesting here.  Aren't you saying to enable debug logging in the config which is what the message advises?

Reply:
Yes. But it is optional in plugin code.
It should be in documentation.